test/integration/test/apply_test.go

```diff
@@ -467,6 +467,9 @@ func TestApply(t *testing.T) {
 		testNodes(t, numMasters(machines), numWorkers(machines))
 	})
 
+	t.Log("Waiting 1 minute for nodes to settle")
+	time.Sleep(1 * time.Minute)
```

Comment:
I added this because I thought it would show the failure I tripped over.
But nothing failed - the controller won't update a master node if there are fewer than two available.
Probably we should take it back out again to allow tests to complete faster.